feature-detects/scroll-to-options.js

```diff
@@ -0,0 +1,27 @@
+/*!
+{
+  "name": "scrollToOptions dictionary",
+  "property": "scrolltooptions"
+}
```

Comment:
coudl you add at least the tags for author (yourself :-) to this? also if there is a caniuse property check for this add this too as well as other links that might be useful.